airmozilla/starred/models.py

```diff
@@ -0,0 +1,12 @@
+from django.core.cache import cache
+from django.db import models
+from django.contrib.auth.models import User
+from airmozilla.main.models import Event
+
+
+class StarredEvent(models.Model):
+    event = models.ForeignKey(Event)
+    user = models.ForeignKey(User)
+    created = models.DateTimeField(auto_now_add=True)
+
```

Comment:
We should put up an additional guard that prevents there from ever existing duplicates here that only differ in their `created` value. Check out the django documentation on how to use `unique_together` in a `Meta` class on the model.